qiskit/synthesis/two_qubit/two_qubit_decompose.py

```diff
@@ -1572,3 +1592,8 @@ def num_basis_gates(self, unitary):
 
 
 two_qubit_cnot_decompose = _LazyTwoQubitCXDecomposer()
+"""
+This is an instance of :class:`.TwoQubitBasisDecomposer` that always uses
+``cx`` as the KAK gate for the basis decomposition. You can use this function
+as a quick access to ``cx``-based 2-qubit decompositions.
+"""
```

Comment:
Nice! We can also add the arguments too:

```suggestion
as a quick access to ``cx``-based 2-qubit decompositions.

Args:
        basis_fidelity (float): Fidelity to be assumed for applications of KAK Gate. Default is 1.0.
        euler_basis (str): Basis string to be provided to :class:`.OneQubitEulerDecomposer` for 1Q synthesis.
            Valid options are ``['ZYZ', 'ZXZ', 'XYX', 'U', 'U3', 'U1X', 'PSX', 'ZSX', 'RR']``.
        pulse_optimize (None or bool): If ``True``, try to do decomposition which minimizes
            local unitaries in between entangling gates. This will raise an exception if an
            optimal decomposition is not implemented. Currently, only ``[{'CX', 'SX', 'RZ'}]`` is known.
            If ``False``, don't attempt optimization. If ``None``, attempt optimization but don't raise
            if unknown.
"""
```